pages/06.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md

```diff
@@ -0,0 +1,68 @@
+|**Modify contact’s segments**| Adds or removes Contacts to/from Segments. If a Contact is removed from a dynamic (filter-based) Segment by a Campaign action, they won’t be re-added to the Segment based on meeting the filter criteria.|
+|**Modify contact’s tags**| Overwrites or appends Tags on a Contact record. You can add or remove Tags, or do both, in the same action.|
+|**Push contact to integration**| Sends the Contact record to the r> selected integration, either creating a new Contact in the chosen integration or updating the connected Contact record.|
+|**Remove Do Not Contact**| Removes the user from the Do Not Contact (DNC) list.|
+|**Send a webhook**| Sends a Webhook to a defined URL, using the GET, POST, PUT, PATCH, or DELETE methods. Headers and data are customizable, and support the use of tokens, such as contact fields and the contact's IP address. For example, {contactfield=firstname}|
+|**Send email**| Sends a transaction or marketing Email to the selected contact. A transactional Email can be sent to the contact multiple times. A marketing Email can be sent to the contact only once across multiple sources. If the contact has already received this Email from another source or the current Campaign, the Email will not be sent again and the contact progresses through the Campaign.|
+|**Send email to user**| Sends an Email to an entity other than the contact. This may be a Mautic user, the Contact’s owner, or non-users. 
+
+>>> Emails sent using this action do not generate any statistics for Contacts or Emails.|
+|**Send marketing message**| Sends a message using the Contact's preferred Channel.|
+|**Update contact**| Updates the existing Contact's fields with the specified values.|
+|**Update contact's primary company**| Updates the existing Contact's primary Company fields with the specified value. See [documentation on Companies][companies].|
+|**Update contact owner**| Updates the Contact's owner.|
+|
+
+
+**Notes**:
+1.  As the first step of your Campaign, you typically send out an Email to your Segments. When you add an Email to a Campaign, you can select a potential **delay** for when the Email is delivered as shown in the following image.
+
+ If the action is attached to a decision's **non-action** initiated decision path, the delay becomes how long the Contact has to take action before the Campaign progresses down the non-action path. 
+<br>
+![Image showing delayed actions on a non-action decision path in a campaign](send-email-delay-nonaction.png)
+
+2. The Delete Contact action also deletes the Campaign event log record about that contact. Therefore, though this action might always display 0% progress on the Campaign detail page, it could have deleted some Contacts.
+
+   The Delete Contact action doesn't allow other Campaign events to be connected to it. Since the Contact will not exist after this action is triggered, Campaign events cannot be triggered after this point.
+
```

Comment:
```suggestion
```

Reply:
This comment was on an older file that was merged. This file doesn't exist any longer.